src/routes/github/create-branch/github-branches-get.ts

```diff
@@ -0,0 +1,40 @@
+import { Request, Response } from "express";
+import { createUserClient } from "~/src/util/get-github-client-config";
+
+export const GithubBranchesGet = async (req: Request, res: Response): Promise<void> => {
+	const { githubToken, jiraHost, gitHubAppConfig } = res.locals;
+
+	if (!githubToken || !jiraHost || !gitHubAppConfig) {
```

Comment:
We can pull this out `|| !jiraHost` we can't guarantee we will have it :( so we've set it optional everywhere